packages/office-ui-fabric-react/src/utilities/positioning/positioning.ts

```diff
@@ -877,6 +867,15 @@ export function positionCallout(
   return _positionCallout(props, hostElement, elementToPosition, previousPositions);
 }
 
+export function positionCard(
```

Comment:
Why are creating a public `positionCard` and a private `positionCard` method here?

Reply:
Because then we can do whatever we want to change the private position card and keep entirely out of the api.